App/src/main/res/values/strings.xml

```diff
@@ -88,5 +88,6 @@
     <string name="required">Required.</string>
     <string name="place_name_already_in_use">Name is already in use</string>
     <string name="add_place">Add place</string>
+    <string name="gps_disabled">Your GPS seems to be disabled, do you want to enable it?</string>
```

Comment:
Yes – do we use low accuracy (like WIFI, iirc) location access, though, or only high accuracy ones?